Quiz

mattbruton opened the pull request:
Here's the quiz. Had some issues getting the conditional statement to work properly for a while, but managed to figure out something to at least behave in a way that passes the requirements. I'm sure there is a more succinct way to code it.

mattbruton committed:
added index.html and quiz.js files

mattbruton committed:
have tree working in console, need to add inputs to control tree height and character selection

mattbruton committed:
editted treeString line of makeTreeString function

mattbruton committed:
improved comments in js file

mattbruton committed:
removed css file used for testing

mattbruton committed:
editted html to reflect removed css file

mattbruton committed:
inputs functional, just need if statement for alert

mattbruton committed:
completed first working draft of quiz

mattbruton committed:
removed event handler from html and added to function in javascript file

mattbruton committed:
removed unnecessary div tags from html

mattbruton committed:
fixed conditional statement in javascript file

mattbruton committed:
editted comments in .js file

JoeShep commented:
Matt, this works exactly as required, except for one thing: The user needs to be able to grow the tree by clicking the button or by hitting the return/enter key. I know the instrux might look like you had the choice, but you need to do both so the user has the choice. So, just add that in and you're good to go!

mattbruton committed:
added submit on enter, also fixed code to pass the object in the function, not setting the function up exclusivly for that particular object

mattbruton commented:
Fixed, also rewrote the code for the function that builds the tree in console to allow treeObject to be passed into it instead of the function looking for treeObject.height or .character (even though you didn't ask!)

mattbruton committed:
removed unnecessary js

mattbruton committed:
removed more useless JS

mattbruton committed:
moved spacing variable to function

zoeames commented:
:+1:
